### PR TITLE
fix: fix Safari display problems

### PR DESCRIPTION
**Pull request by thevoiceofzeke**
[MUMUP-3138](https://jira.doit.wisc.edu/jira/browse/MUMUP-3138): "The following widget types don't display the contents of the Widget configuration text area in widget creator on Safari 11:
List of links
Search with links
Action Items"

**In this PR:**
- Upgrade to uportal-app-framework 7.0.0
- Can no longer reproduce bug in Safari 11

### Demo
safari-fix

### CODE REVIEW

**Review by vertein — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -55,7 +55,7 @@
     <dependency>
       <groupId>org.apereo.uportal</groupId>
       <artifactId>uportal-app-framework</artifactId>
-      <version>6.1.1-SNAPSHOT</version>
+      <version>7.0.0</version>
```
Comment (apetro) [inline pom.xml]: 👍 . Was Widget Creator deliberately using a `SNAPSHOT` so as to be another canary in the mine about breaking framework changes? Worth using `7.0.1-SNAPSHOT` here for that continued canary?

**Review by apetro — APPROVED** [review]

**Review by ChristianMurphy — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (thevoiceofzeke) [reply pom.xml]: Not sure why it was using a `SNAPSHOT`. I will defer to whoever has an opinion about this.

Comment (davidmsibley) [reply pom.xml]: :+1: to using SNAPSHOT. This way we get the latest running on the test tier. I think I changed it to SNAPSHOT the last time I was working on it.

Comment (davidmsibley) [reply pom.xml]: Also, since we do manual interactive releases of this, there's no real danger of keeping this on SNAPSHOT

Comment (thevoiceofzeke) [reply pom.xml]: Done

**Review by davidmsibley — APPROVED** [review]

**Pull request merged**